src/logger/logger.rs

```diff
@@ -0,0 +1,36 @@
+use simplelog::*;
+use std::fs::File;
+
+fn get_log_level(env_var_name: &str, default_level: LevelFilter) -> LevelFilter {
+    match std::env::var(env_var_name) {
```

Comment:
You should inject the configuration wrapper inside the function as a param instead of directly accessing the data 
```suggestion
fn get_log_level(config: &str, default_level: LevelFilter) -> LevelFilter {
    match config.bind<String>("< the key you want to use inside the json config>") {
```

By doing this you allow the logger to manage its own configuration without having to modify the env name outside of the function

Reply:
Agree, I'll implement this